From: Sven Roederer <devel-sven@geroedel.de>
Date: Fri, 30 Apr 2021 21:54:22 +0200
Subject: Revert "opkg: update to git HEAD"

This reverts commit 988ed0080284903d1fe4851c5ae8f1238bc61da2 which bumps opkg
to 5936c4f. this change is expected to cause errors in looking up dependencies
of packages installed from local filesystem.

see http://lists.openwrt.org/pipermail/openwrt-devel/2021-April/034887.html

diff --git a/package/system/opkg/Makefile b/package/system/opkg/Makefile
index fb7d8667c66e6d0ed9e63bba7a262828ba7e169d..6c3682571181fb45211d7faf63c59df8c7868373 100644
--- a/package/system/opkg/Makefile
+++ b/package/system/opkg/Makefile
@@ -14,9 +14,9 @@ PKG_FLAGS:=essential
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://git.openwrt.org/project/opkg-lede.git
-PKG_SOURCE_DATE:=2021-03-15
-PKG_SOURCE_VERSION:=5936c4f9660248284e8a9b040ea3153d3ea888de
-PKG_MIRROR_HASH:=b873c209baaf4f150c89646d58e4a0072f807d24b02c320ab8c7ae9180c13240
+PKG_SOURCE_DATE:=2021-03-14
+PKG_SOURCE_VERSION:=d3a63b327383a2ac1a8f925a58a2f6a7808f5181
+PKG_MIRROR_HASH:=65339e965b3d3d2d49b98e41d774bcedb0f044335c651d306d95cd3e96ca57c3
 
 PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=COPYING
